pymc3/distributions/logprob.py

```diff
@@ -342,9 +342,24 @@ def subtensor_logp(op, var, rvs_to_values, indexed_rv_var, *indices, **kwargs):
     return logp_var
 
 
-def logcdf(*args, **kwargs):
+def logp(var, rv_values, **kwargs):
+    """Create a log-probability graph."""
+
+    # Attach the value_var to the tag of var when it does not have one
+    if not hasattr(var.tag, "value_var"):
```

Comment:
This is needed to avoid a warning issued in `aesaraf::rvs_to_value_vars` function.